File: ReadingList.ReadModel/Models/BookList/PrivateBookList/PrivateBookList.cs
using System.Collections.Generic;

namespace ReadingList.ReadModel.Models
{
    public class PrivateBookList : BookList
    {
        public PrivateBookList()
        {
            Items = new List<PrivateBookListItem>();
        }

        public List<PrivateBookListItem> Items { get; set; }
    }
}

File: ReadingList.Client/src/models/BookList/Implementations/PrivateBookList.ts
import { List } from '..';
import { PrivateBookListItem } from './PrivateBookListItem';

export class PrivateBookList implements List<PrivateBookListItem> {
    isInEditMode: boolean = false;
    name: string;
    id: number;
    items: PrivateBookListItem[];
    type: number;
}